public/app/features/plugins/partials/ds_http_settings.html

```diff
@@ -64,6 +64,7 @@ <h3 class="page-heading">Auth</h3>
   <div class="gf-form-group">
     <div class="gf-form-inline">
       <gf-form-switch class="gf-form" label="Basic Auth" checked="current.basicAuth" label-class="width-8" switch-class="max-width-6"></gf-form-switch>
+      <gf-form-switch class="gf-form" label="Custom Headers" checked="current.addCustomHeaders" label-class="width-8" switch-class="max-width-6"></gf-form-switch>
```

Comment:
the width for the gf-form-switch should be 11. and if should be last in the list of form-switches.